virttest/test_setup.py

```diff
@@ -8,6 +8,8 @@
 import random
 import math
 import shutil
+import arch
+from netaddr import *
```

Comment:
@xutian I have updated requirements.txt.. same prereq package installation is going through fine on my x86 laptop, but seeing travis-ci reporting issues :-(